docs/advanced/directories.rst

```diff
@@ -0,0 +1,28 @@
+.. _directories:
+
```

Comment:
This is a great start, but needs some editing / wordsmithing improvements.

Reply:
@michaeljoseph Is there anything in particular that you think is missing, or should I just make it generally more verbose?

Reply:
I've made some suggestions...